packages/first-boot-wizard/files/usr/lib/lua/firstbootwizard.lua

```diff
@@ -161,27 +164,32 @@ function fbw.fetch_config(data)
     local hostname = utils.execute("wget --no-check-certificate http://["..data.host.."]/cgi-bin/hostname -qO - "):gsub("\n", "")
     fbw.log('Hostname found: '.. hostname)
     if (hostname == '') then hostname = host end
-    local signal = data.signal
-    local ssid = data.ssid
-    local filename = fbw.WORKDIR .. fbw.HOST_CONFIG_PREFIX .. hostname
 
-    utils.execute("wget --no-check-certificate http://[" .. data.host .. "]/cgi-bin/lime/lime-community -O " .. filename)
-    if not utils.file_exists(filename) then
+    local lime_community_fname = fbw.WORKDIR .. fbw.COMMUNITY_HOST_CONFIG_PREFIX .. hostname
+
+    utils.execute("wget --no-check-certificate http://[" .. data.host .. "]/cgi-bin/lime/lime-community -O " .. lime_community_fname)
```

Comment:
it is odd that wget uses ` --no-check-certificate` when the request is http... not from this commit, but still :)

Reply:
Indeed, that is because uhttpd may have "redirect to https" enabled with a self signed certifcate (check this PR https://github.com/libremesh/lime-packages/pull/799)

Reply:
Thanks! was not aware of that!